src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -93,4 +98,39 @@ def bs_request
       @notification.notifiable.commentable
     end
   end
+
+  def link_for_reportables
```

Comment:
I think we are duplicating *a lot* of decisions from `notifiable_link_text` here.  Guess we should also to per reportable STIs. But not now... https://trello.com/c/hWSLOVRl